docs/spec/v1beta2/alerts.md

```diff
@@ -0,0 +1,183 @@
+# Alerts
+
+The `Alert` API defines how events are filtered by severity and involved object, and what provider to use for dispatching.
+
+## Example
+
+The following is an example of how to send alerts to Slack when Flux fails to reconcile the `flux-system` namespace.
```

Comment:
I think providing a working example of a real use-case is what that first section should be about. If we did something in SC that doesn't make it a standard set in stone. Instead of "Working with Alerts" here is just the first example, for Providers, instead of "Working with Provider" is called  "Provider guides".

Reply:
We agreed on unifying the titles to help people navigate the pages better, as they will be uniform in terms of navigation. The source-controller document structure has been derived from Kubernetes documentation, which again makes it easier for people to navigate the pages if they're familiar with the Kubernetes core documentation.

I also do not see how the information under "Provider guides" couldn't be reshaped into something that would work like "Working with Providers". I actually think that for the time being, a title change (almost) would be sufficient.

Reply:
I've rename that section in both providers and receivers.